From 3e1367b680b26123e9e1aa09f8caf2a932d47572 Mon Sep 17 00:00:00 2001
From: Lars Op den Kamp <lars@opdenkamp.eu>
Date: Tue, 17 Apr 2012 01:57:43 +0200
Subject: [PATCH 2/2] cec: libCEC 1.6 support and fixes. libCEC 1.6+ is needed
 when using firmware v2 on the CEC adapter, which adds
 wake over CEC   * added a new setting to control
 whether to put the TV in standby when the player is put
 in standby.   * added some button mappings: all menu
 related buttons -> menu, previous channel -> teletext,
 added support for the channels list on samsung, mapped
 next fav -> menu (when available)   * display the
 firmware version in the peripheral manager (if
 available)   * handle the new CEC alert callback
 (libCEC 1.6+)   * replaced 'Put this PC in standby mode
 when the TV is switched off' with an enum that allows
 the user to chose between 'Ignore', 'Suspend' and
 'Shutdown'   * fixed - crash when changing settings
 without libCEC started.   * fixed - range of wake and
 power-off devices   * fixed - update the correct
 standby device setting ('standby_devices' not
 'wake_devices')   * fixed - don't get the settings from
 the eeprom, but always use the settings provided in
 xbmc

---
 .../720p/DialogPeripheralManager.xml               |    4 +-
 configure.in                                       |    2 +-
 language/English/strings.xml                       |    5 +-
 project/BuildDependencies/scripts/libcec_d.txt     |    2 +-
 system/peripherals.xml                             |   15 +-
 tools/darwin/depends/libcec/Makefile               |    2 +-
 xbmc/peripherals/bus/PeripheralBus.cpp             |    1 +
 xbmc/peripherals/devices/Peripheral.h              |    2 +
 xbmc/peripherals/devices/PeripheralCecAdapter.cpp  |  160 ++++++++++++++++----
 xbmc/peripherals/devices/PeripheralCecAdapter.h    |    4 +
 10 files changed, 152 insertions(+), 45 deletions(-)

diff --git a/720p/DialogPeripheralManager.xml b/720p/DialogPeripheralManager.xml
index 07954d0..5717017 100644
--- a/720p/DialogPeripheralManager.xml
+++ b/720p/DialogPeripheralManager.xml
@@ -117,7 +117,7 @@
 						<posy>50</posy>
 						<width>520</width>
 						<height>20</height>
-						<label>$LOCALIZE[35501]:  [COLOR=grey2]$INFO[ListItem.Property(Class)][/COLOR]</label>
+						<label>$LOCALIZE[35501]:  [COLOR=grey2]$INFO[ListItem.Property(Class)][/COLOR]     $LOCALIZE[19114]:  [COLOR=grey2]$INFO[ListItem.Property(Version)][/COLOR]</label>
 						<align>left</align>
 						<aligny>center</aligny>
 						<font>font12</font>
@@ -173,7 +173,7 @@
 						<posy>50</posy>
 						<width>520</width>
 						<height>20</height>
-						<label>$LOCALIZE[35501]:  [COLOR=grey2]$INFO[ListItem.Property(Class)][/COLOR]</label>
+						<label>$LOCALIZE[35501]:  [COLOR=grey2]$INFO[ListItem.Property(Class)][/COLOR]     $LOCALIZE[19114]:  [COLOR=grey2]$INFO[ListItem.Property(Version)][/COLOR]</label>
 						<align>left</align>
 						<aligny>center</aligny>
 						<font>font12</font>
-- 
1.7.5.4

